### ISSUE TITLE
Replace 5 with $quality - Else quality is always 5

### ISSUE THREAD

**Issue by jenstornell** [jenstornell/php-image-converter#10]
It should probably be `$quality` instead of `5`.

```php
function convert($from, $to, $quality = -1) {
  $converter = new ImageConverter();
  return $converter->convert($from, $to, 5);
}
```

**Comment by peter279k** [comment]
It's not weird.

- According to the `imagejpeg` function [reference](https://www.php.net/manual/en/function.imagejpeg.php), the default image quality value is `-1`.
- According to the `imagepng` function [reference](https://www.php.net/manual/en/function.imagepng.php), the default image quality value is `-1`.
- The [imagegif ](https://www.php.net/manual/en/function.imagegif.php)and [imagebmp ](https://php.net/manual/en/function.imagebmp.php)functions don't have the `quality` value.

Could you explain more details about why you think this default quality is weird?

**Comment by jenstornell** [comment]
@peter279k I was in a hurry when adding this issue. It's not the built in PHP default value that is weird, it's my convert function default value that is weird, or wrong is a better word.

When sending a `$quality` value to it, it will be ignored, because it's never sent to the class method. It will always end up as `5`.

I just need to move the `$quality` argument and replace `5` with it and it should be fine.

**Comment by peter279k** [comment]
@jenstornell, thank you for your explanation.

I know that and I think we have to consider/note that some image formats in PHP `GD` extension will not have the `quality` argument.

**Comment by peter279k** [comment]
And the function you create should change into following code snippets:

```php
function convert($from, $to, $quality = -1) {
  $converter = new ImageConverter();
  return $converter->convert($from, $to, $quality);
}
```

**Comment by jenstornell** [comment]
@peter279k Yes. I tested sending a quality argument to a method that did not support it and it will just be ignored, so I think it will be fine out of the box. I think a note about it in the docs will do (maybe that's what you ment?). Also the quality argument differ between formats. Some have 1-100 and others 1-9 etc. Docs would be good not forcing users to go to php.net.

Yes, I know how the function will look like. I did not have the time to implement it yesterday, that's why I added an issue instead.
